src/classes/GE_DefaultTemplate.cls

```diff
@@ -171,43 +202,43 @@ public with sharing class GE_DefaultTemplate {
         newDefaultTemplate.Format_Version__c = TEMPLATE_VERSION;
         return newDefaultTemplate;
     }
-
+    
     /**
      * @description This method builds a mapping of partial dev name to 
      * the full dev name using the org's field mapping custom metadata records. 
      *
      * @return A mapping of partial dev name to full dev name. 
      * e.g. Account1_Name -> Account1_Name_ee63cf9df
      */
-    public static Map<String, String> getCurrentDevNameMapping() {
+    public static void populateDevNameMapping(Map<String,String> updatedFieldDevNameMap, 
+        Map<String,String> updatedObjectDevNameMap) {
+
         // Get the org's migrated field mappings. 
-        List<Data_Import_Field_Mapping__mdt> fieldMappingCMTList = 
-            [SELECT Id, 
-                    MasterLabel, 
-                    DeveloperName,
-                    Target_Field_API_Name__c, 
-                    Source_Field_API_Name__c,
-                    Target_Object_Mapping__r.Object_API_Name__c
-            FROM Data_Import_Field_Mapping__mdt 
-            WHERE Data_Import_Field_Mapping_Set__r.DeveloperName =: 
-                BDI_MigrationMappingUtility.MIGRATED_CUSTOM_FIELD_MAPPING_SET_RECORD_NAME 
-            LIMIT 1000];
-
-        // A map which contains the partial dev name to the org's full dev name.
-        // An example would like this:
-        // Account1_Name -> Account1_Name_ee63cf9df
-        Map<String, String> partialDevNameToUpdatedDevName = new Map<String, String>();
-        
-        for (Data_Import_Field_Mapping__mdt fieldMapping : fieldMappingCMTList) {
-            String partialDevName = fieldMapping.DeveloperName.substringBeforeLast('_');
+        List<Data_Import_Field_Mapping__mdt> fieldMappingCMTList = [
+                SELECT Id, 
+                        MasterLabel, 
+                        DeveloperName,
+                        Target_Object_Mapping__r.DeveloperName
+                FROM Data_Import_Field_Mapping__mdt 
+                WHERE Data_Import_Field_Mapping_Set__r.DeveloperName =: 
+                    BDI_MigrationMappingUtility.MIGRATED_CUSTOM_FIELD_MAPPING_SET_RECORD_NAME
+            ];
 
+        for (Data_Import_Field_Mapping__mdt fieldMapping : fieldMappingCMTList) {
+            String partialFieldDevName = fieldMapping.DeveloperName.substringBeforeLast('_');
+			String partialObjectDevName = 
```

Comment:
Just a note that the indentation here looks off in GitHub but looks totally fine in my vs code?